### PR TITLE
Update RestClient to ~> 2.0.1

### PR DESCRIPTION
**Pull request by quigebo**
Bump the RestClient version from ~> 1.6 to ~> 2.0.1

Our app currently uses RestClient 2.0.1, so Bundler wasn't able to install the Sonar gem. We obviously would prefer to use the official Ruby gem versus writing our own client.

### CODE REVIEW

File: send_sonar.gemspec [hunk 1]
```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "vcr", "~> 2.6"
   spec.add_development_dependency "webmock", "~> 1.1"
   spec.add_development_dependency "json"
+  spec.add_development_dependency "pry"
```
Comment (AlexDWu) [inline send_sonar.gemspec]: Why is this necessary?

File: lib/send_sonar/exceptions.rb [hunk 2]
```diff
@@ -24,7 +24,7 @@ def initialize(original_exception)
     end
 
     def inspect
-      @original_exception.inspect
+      "#{@original_exception.to_s}: #{@original_exception.http_body}"
```
Comment (AlexDWu) [inline lib/send_sonar/exceptions.rb]: Is this really necessary?

File: send_sonar.gemspec (hunk 1 above)
Comment (quigebo) [reply send_sonar.gemspec]: oops. totally right

File: lib/send_sonar/exceptions.rb (hunk 2 above)
Comment (AlexDWu) [reply lib/send_sonar/exceptions.rb]: Wait, "match specs"?

**Comment by greysteil** [comment]
Not 100% sure it's relevant here, but you might want to consider using a range for `rest-client` - something like `">= 1.6", "< 3.0"`. I build [Dependabot](https:://dependabot.com) to automatically update Gemfile / gemspec requirements when new versions are released, and that's the approach we take for gemspecs in order to maximise compatibility.
